mqtt/mqttd/src/time.rs

```diff
@@ -1,4 +1,4 @@
-//! `tokio::time::delay_for` and `tokio::time::delay_until` has a bug that
+//! `tokio::time::sleep` and `tokio::time::delay_until` has a bug that
```

Comment:
is this still true?

Reply:
good catch! seems like it was fixed recently.

Reply:
2.2y is not enough for us?

Reply:
for test purposes I made the one for 10y and it caused some problems for me previously.